lms/djangoapps/badges/events/course_complete.py

```diff
@@ -32,7 +32,11 @@ def course_slug(course_key, mode):
     Badgr's max slug length is 255.
     """
     # Seven digits should be enough to realistically avoid collisions. That's what git services use.
-    digest = hashlib.sha256(u"{}{}".format(six.text_type(course_key), six.text_type(mode))).hexdigest()[:7]
+    slug = u"{}{}".format(six.text_type(course_key), six.text_type(mode))
+    if six.PY3:
+        slug = slug.encode('utf-8')
```

Comment:
I think you should always do this.  Since in python2 it was being done implicitly and doing it explicitly won't break anything.